src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -127,13 +127,19 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
     return this._defaultCountry;
   }
 
-  /**
-   * Specifies the format for validated phone numbers.
+  /* Specifies the format for validated phone numbers.
```

Comment:
I think you accidentally removed the top `/**` from the JSDocs comment. This will prevent it from being read properly by TypeDoc.

Reply:
Fixed